_posts/2025-04-16-#31689.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: pr
+date: 2025-04-16
```

Comment:
I saw that you put April 2nd on the scheduling sheet but April 16 here - is this intentional?

Reply:
Oops, fixed that, thanks!